src/hawkmoth/parser.py

```diff
@@ -401,6 +401,9 @@ def _get_scopedenum_type(cursor):
             return f': {cursor.enum_type.spelling}'
     return None
 
+def _convert_bool(type_string):
```

Comment:
I suggest a more generic `_normalize_type` or something. It reads better in its call site, the fact that it only touches the bool case is irrelevant there.

Reply:
Agreed.

Reply:
Fixed.